### PR TITLE
Add disclaimer on users page

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _data/users.yml [hunk 1]
```diff
@@ -1,3 +1,11 @@
+################################################################################
+# Users Page
+#
+# Disclaimer: The companies are listed here for their usage of Trino. Inclusion
```
Comment (jhlodin) [inline _data/users.yml:4]: Edit: "The following companies are listed..."

File: _data/users.yml [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+# of a marquee on the users page does not imply an endorsement of that company
```
Comment (jhlodin) [inline _data/users.yml:5]: "on the users" -> "on this users"

File: _data/users.yml [hunk 3]
```diff
@@ -1,3 +1,11 @@
+################################################################################
+# Users Page
```
Comment (mosabua) [inline _data/users.yml:2]: Data for the users display on the home page and the detailed users page:

https://trino.io
https://trino.io/users.html

File: _data/users.yml [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
+#
+# Disclaimer: The following companies are listed here for their usage of Trino.
```
Comment (mosabua) [inline _data/users.yml:4]: Remove "Disclaimer:"

Drop "here"

File: _data/users.yml [hunk 5]
```diff
@@ -1,3 +1,11 @@
...
+# Inclusion of a marquee on this users page does not imply an endorsement of
```
Comment (mosabua) [inline _data/users.yml:5]:
```suggestion
# Inclusion of a marquee on the Trino site does not imply an endorsement of
```

**Review by mosabua — CHANGES_REQUESTED** [review]

**Review by martint — APPROVED** [review]

File: _data/users.yml [hunk 6]
```diff
@@ -1,3 +1,14 @@
+################################################################################
+# Data for the users display on the home page and the detailed users page:
+#
+# * https://trino.io
+# * https://trino.io/users.html
+#
+# The following companies are listed for their usage of Trino. Inclusion of a
+# marquee on the Trino Site does not imply an endorsement of that company from
```
Comment (mosabua) [inline _data/users.yml:8]: site

**Review by mosabua — APPROVED** [review]

**Pull request merged**